_data/compatibility/blocksettle.yaml

```diff
@@ -0,0 +1,96 @@
+---
+name: BlockSettle
+internal_url: /en/compatibility/blocksettle
+logo: /img/compatibility/blocksettle/blocksettle.png
+rbf:
+  tested:
+    date: "2020-10-29"
+    platforms:
+      - Windows
+      - macOS
+      - Linux
+    version: "0.91.1"
+  features:
+    receive:
+      notification: "true"
+      list: "true"
+      details: "true"
+      shows_replaced_version: "true"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "true"
+      list: "true"
+      details: "false"
+      shows_replaced_version: "true"
+      shows_original_version: "false"
+  examples:
+    - image: /img/compatibility/blocksettle/rbf/RBF_desktop_notification.png
+      caption: >
+        Desktop notification includes RBF signalling
```

Comment:
Suggestion to replace signalling with signaling as we standardized on that in the rest of the matrix.